ts/boot/configurePushNotification.ts

```diff
@@ -29,6 +40,28 @@ const NotificationPayload = t.partial({
   })
 });
 
+/**
+ * Decide how to refresh the messages based on pagination.
+ */
+function handleMessageReload() {
+  const cursors = getCursors(store.getState());
+  if (pot.isNone(cursors)) {
+    // nothing in the collection, refresh
+    store.dispatch(reloadAllMessages.request({ pageSize }));
+  } else if (pot.isSome(cursors)) {
+    // something in the collection, get the new ones only
+    store.dispatch(
+      loadPreviousPageMessages.request({
+        cursor: cursors.value.previous,
+        pageSize: 100
```

Comment:
```suggestion
        pageSize
```
maybe we should use the config const `pageSize` defined in `ts/config.ts`